src/features/student/queries/use-student-summary.query.ts

```diff
@@ -0,0 +1,25 @@
+import type { BaseApiException } from '@/common/exceptions/base-api.exception'
+import type { StudentSummaryDTO } from '@/types'
+import profile_picture_banner from '@/assets/profile_banner_placeholder.png'
+import profile_picture_placeholder from '@/assets/profile_picture_placeholder.png'
+import { useQuery, type UseQueryReturnType } from '@tanstack/vue-query'
+
+function useStudentProfileSummaryQuery (): UseQueryReturnType<StudentSummaryDTO, BaseApiException> {
```

Comment:
```suggestion
function useStudentSummaryQuery (): UseQueryReturnType<StudentSummaryDTO, BaseApiException> {
```

On récupère un `StudentSummaryDTO` donc il faudrait que le nom reflète ce qu'on récupère. Sinon il faudrait que le back appelle ça `StudentProfileSummaryDTO`